interp/declare.ml

```diff
@@ -126,7 +126,7 @@ let (inConstant : constant_obj -> obj) =
   declare_object { (default_object "CONSTANT") with
     cache_function = cache_constant;
     load_function = load_constant;
-    open_function = open_constant;
+    open_function = import_filter [] open_constant;
```

Comment:
>Just in case my comment was not clear, I'd like to see the list of categories be part of the object itself (and default_object could have an empty one).

Please read the PR header message again.